pyproject.toml

```diff
@@ -70,7 +70,7 @@ exclude = [
 [tool.coverage.run]
 relative_files = true
 source = ["."]
-omit = ["tests/**", "docs/**", "lib/**", "snap/**", "build/**", "setup.py"]
+omit = ["tests/**", "docs/**", "lib/**", "snap/**", "build/**", "setup.py", "cou/zaza_utils/**"]
```

Comment:
Why do not use lib directory for zaza?

Reply:
That's why I think we should attached hash of zaza version, so we can easily see which version was used. Using `lib` directory is just more intuitive for me instead of hiding it in source code, but yes these is not sub-module so no strong opinion.